src/components/HeadOfTable/HeadOfTable.tsx

```diff
@@ -0,0 +1,38 @@
+import classNames from 'classnames';
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+export const HeadOfTable: React.FC = () => {
```

Comment:
I think this component should be named as `Navigation` and `Navigation` should be `PageLayout` or `MainLayout`.

P.S Currently you render two navigation

![image](https://github.com/mate-academy/react_people-table-basics/assets/96138503/d7c91808-ffd7-4ae9-95ec-57f873a48b3f)